app/Controller/CoGroupsController.php

```diff
@@ -465,6 +555,19 @@ function isAuthorized() {
     // View an existing Group?
     $p['view'] = ($roles['cmadmin'] || $roles['coadmin'] || $managed);
     
+    // Viewing members, email lists, and nested groups is available to all group members.
+    // Access to specific actions is controlled in the view.
+    $p['members'] = $roles['cmadmin'] || $roles['coadmin'] || $managed || $member;
+    $p['email_lists'] = $roles['cmadmin'] || $roles['coadmin'] || $managed || $member;
+    $p['nest'] =  $roles['cmadmin'] || $roles['coadmin'] || $managed || $member;
```

Comment:
I'm not sure `|| $member` is appropriate here. The only permission that was previously intended was for a member to see their own membership (ie: the `co_group_member`) view, and maybe that wasn't even fully thought out.

Reply:
Yeah - this is worth a discussion. We can definitely scale this back - though it does beg the question of what a member sees when they visit a group they are in.

Reply:
See [comment](https://github.com/Internet2/comanage-registry/pull/185#issuecomment-910264336) down below